django/cantusdb_project/main_app/tests/make_fakes.py

```diff
@@ -140,8 +142,12 @@ def make_fake_chant(
         c_sequence = random.randint(1, MAX_SEQUENCE_NUMBER)
     if cantus_id is None:
         cantus_id = make_random_string(6, "0123456789")
+    if image_link is None:
+        image_link = faker.image_url()
     if feast is None:
         feast = make_fake_feast()
+    if mode is None:
+        mode = make_random_string(1, "0123456789*?")
```

Comment:
Just want to note that the mode field could have different values that these here and could be 2 characters long...that may not be material here, but since you are doing ordering it may!